### PR TITLE
Added a new quote

### PR DESCRIPTION
**Pull request by epletnev**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order by author's first name and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -103,6 +103,10 @@
     "author": "Benjamin Whorf",
     "sentence": "Language is not simply a reporting device for experience but a defining framework for it."
   },
+  {
+    "author": "Bob Dylan",
+    "sentence": "A mistake is to commit a misunderstanding."
```
Comment (Kvaibhav01) [inline server/data/quotes.json:108]: Please add a quote regarding "learning a foreign language" somewhat similar.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR. Just update the changes.

File: server/data/quotes.json (hunk 1 above)
Comment (epletnev) [reply server/data/quotes.json]: I have changed the quote

**Review by jayehernandez — APPROVED** [review]
Thank you for contributing to Letra ☀️, @epletnev!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**